tests/integration/lxd/test_launcher.py

```diff
@@ -267,7 +268,7 @@ def test_launch_use_base_instance(
         .rstrip("\n")
     )
 
-    assert instance_hostname == base_configuration._hostname
+    assert instance_hostname == instance.instance_name
```

Comment:
Because the creation order changed.
Old:
`base_configuration` is the target instance hostname, and we rename it to the base instance hostname later.
New:
`base_configuration` is the base instance hostname, and we rename it to the target instance hostname later.

The final results did not change.